Client-Side/shecodesprogramming/src/Components/HomePage/homepage.jsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import './homepage.css';
+
+function Homepage(){
+    return(
+        <div>
+            <div className="MainLayout">
+                <div className="layout">
+                    <div className="sublayout">
+                        <img className="welcome" src="https://kalvium.community/images/home_hero.gif" alt="" />
```

Comment:
Please add alternate text for image tag.

Reply:
I added alternate text could you please review it.